### PR TITLE
Add function to control sleep activation mode for havok physics

### PR DESCRIPTION
**Pull request by FlorentMasson**
https://github.com/BabylonJS/Babylon.js/issues/14810

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14816/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14816/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14816/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14816/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14816/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14816/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14816/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14816/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14816/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)
Important - these might fail sporadically. This is an optional test.

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14816/merge/testResults/webgpuplaywright/index.html

**Comment by FlorentMasson** [comment]
Adding a fix
I'm not sure I can add a "new feature" label myself
Test playground: https://playground.babylonjs.com/#Z8HTUN#880 (the sphere should stay red)

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)
Important - these might fail sporadically. This is an optional test.

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14816/merge/testResults/webgpuplaywright/index.html

**Comment by RaananW** [comment]
> [#Z8HTUN#880](https://playground.babylonjs.com/#Z8HTUN#880)

The sphere does stay red in the snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14816/merge#Z8HTUN#880

Please ignore the WebGL2 tests, there is still a formatting issue that needs to be resolved.

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14816/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)
Important - these might fail sporadically. This is an optional test.

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14816/merge/testResults/webgpuplaywright/index.html

**Review by CedricGuillemet — APPROVED** [review]

File: packages/dev/core/src/Physics/v2/IPhysicsEnginePlugin.ts [hunk 1]
```diff
@@ -346,6 +346,15 @@ export enum PhysicsMotionType {
     DYNAMIC,
 }
 
+/**
+ * Controls the body sleep mode.
+ */
+export enum PhysicsActivationControl {
+    SIMULATION_CONTROLLED,
+    ALWAYS_ACTIVE,
+    ALWAYS_INACTIVE
```
Comment (RaananW) [inline packages/dev/core/src/Physics/v2/IPhysicsEnginePlugin.ts:355]: Formatting issue

```suggestion
    ALWAYS_INACTIVE,
```

**Pull request merged**